README.md

```diff
@@ -30,3 +30,43 @@ The Groq API Cookbook is community-driven and we are always looking for contribu
 If you're interested in contributing to our cookbook, please first check out our [contribution guidelines](https://github.com/groq/groq-api-cookbook/blob/main/CONTRIBUTING.md). We welcome high-quality guides and examples that will help our community and can't wait to see what you cook up! 🧑‍🍳
 
 If there are guides or examples that you'd like to see in the future, feel free to make a suggestion [here](https://github.com/groq/groq-api-cookbook/issues).
+
+## Table of tutorials
+
+### Quickstart
```

Comment:
```suggestion
### Quick Start
```

Reply:
Actually, instead of quick start, we should rename this to "Content Filtering" or "Guardrails" as we'll have more tutorials for Llama Guard soon.